src/sentry/features/handler.py

```diff
@@ -24,7 +26,9 @@ def __call__(self, feature: Feature, actor: User) -> bool | None:
         return self.has(feature, actor)
 
     @abc.abstractmethod
-    def has(self, feature: Feature, actor: User, skip_entity: bool | None = False) -> bool | None:
+    def has(
+        self, feature: Feature, actor: User | RpcUser, skip_entity: bool | None = False
+    ) -> bool | None:
```

Comment:
mypy seems to want all subclass implementations to have the same type signature.